projects/rocprofiler-systems/source/lib/rocprof-sys/library/cpu_freq.cpp

```diff
@@ -268,23 +250,26 @@ setup()
     }
 }
 
-void
-config()
-{
-    component::cpu_freq::configure();
-}
-
 void
 sample()
 {
-    auto _ts = tim::get_clock_real_now<size_t, std::nano>();
+    auto _timestamp = tim::get_clock_real_now<size_t, std::nano>();
 
     auto _rcache = tim::rusage_cache{ RUSAGE_SELF };
     auto _freqs  = component::cpu_freq{}.sample();
 
+    trace_cache::get_buffer_storage().store(
```

Comment:
```suggestion
    // user and kernel mode times are in microseconds
    trace_cache::get_buffer_storage().store(
```
Moving comment since it applies here, as well.